spacetelescope/jwst_magic
# JWSTFGS-200 Add OSS defaults

This PR handles the follow main JWSTFGS-200 fix:
- Add new "Use OSS Defaults" checkbox to the main GUI
- Add `use_oss_defaults` and `catalog_countrate` inputs to the FSW writer section of the code
- When `use_oss_defaults` is True and and the `catalog_countrate` of the guide star is passed into `BuildFGSSteps`, the code will use equations provided by Ed (see the original ticket) to calculate the count rate and threshold that should go into the FSW files.
- Test & Docs have been updated accordingly

And the following bug fixes:
- Change FGS magnitude option for normalization to be allowed to put in any magnitude now that we have the magnitude_to_countrate function in the fgscountrate module
- When "create photometry override file" is clicked, need to grey out choosing the shifted/original image from SG section
- When opening new gui, Shift to ID button in enabled by default, but (0,0) and (0,45) should also be enabled
- Update connection between Star Selector and FSW Writer sections. If FSW is checked, star selector is auto-checked too (but not vice versa)
- A file with all the background star information is now written out (in the format y, x, fgs_mag)
- Fixing a JENKINS failure we've been seeing in the last few PRs. It appears there was some kind of change in jenkins that means our previous method of checking our jenkins status stopped working.
